src/cascadia/TerminalApp/CommandPalette.idl

```diff
@@ -16,11 +17,14 @@ namespace TerminalApp
         Windows.Foundation.Collections.IObservableVector<Command> FilteredActions { get; };
 
         void SetCommands(Windows.Foundation.Collections.IVector<Command> actions);
+        void SetKeyBindings(Microsoft.Terminal.TerminalControl.IKeyBindings bindings);
```

Comment:
Why do we need to update our keybindings in CommandPalette? Is this all for the <kbd>Alt</kbd> KeyUp event?

Reply:
read above.